ui/job-view/details/summary/ActionBar.jsx

```diff
@@ -476,6 +477,7 @@ ActionBar.defaultProps = {
   jobLogUrls: [],
 };
 
-export default withNotifications(
-  withPushes(withSelectedJob(withPinnedJobs(ActionBar))),
-);
+export default connect(
+  null,
+  { notify },
```

Comment:
Is `{ notify }` used in place of `mapDispatchToProps`?

Reply:
Yeah, exactly.  It's kinda cool.  If you pass in an object with the functions keyed/named to match, then it does all the wiring for you.  :)